src/rdma.c

```diff
@@ -669,7 +672,27 @@ static void connRdmaEventHandler(struct aeEventLoop *el, int fd, void *clientDat
     }
 }
 
-static int rdmaHandleConnect(char *err, struct rdma_cm_event *ev, char *ip, size_t ip_len, int *port) {
+static int rdmaKeepaliveTimeProc(struct aeEventLoop *el, long long id, void *clientData) {
+    struct rdma_cm_id *cm_id = clientData;
+    RdmaContext *ctx = cm_id->context;
+    connection *conn = ctx->conn;
+    ValkeyRdmaCmd _cmd = {0};
```

Comment:
Why does `_cmd` have an underscore? The other variables don't.

In some functions it has an underscore and in others it doesn't, like in connRdmaGetFeature.

```suggestion
    ValkeyRdmaCmd cmd = {0};
```

Reply:
Because `static int connRdmaGetFeature(RdmaContext *ctx, struct rdma_cm_id *cm_id, ValkeyRdmaCmd *cmd)` has already use `cmd` as parameter.